rbi/sorbet/t.rbi

```diff
@@ -64,8 +64,6 @@ module T
   sig {params(arg: T.untyped).returns(T.untyped)}
   def self.must(arg); end
 
-  def self.coerce(type); end
```

Comment:
This method doesn't actually exist in the runtime anymore, and there are no references of calling it in pay-server.